### PR TITLE
Update index.html.slim

### PR DESCRIPTION
**Pull request by insectengine**
Removing promo that is a duplicate of the hero.

### CODE REVIEW

**Comment by adelasofia** [comment]
this is ok to test

**Comment by adelasofia** [comment]
retest this please

File: index.html.slim [hunk 1]
```diff
@@ -20,9 +20,9 @@ description: The simple, modern & productive way to build apps and infrastructur
         a.spotlight-item href='#{site.base_url}/summit-is-for-developers/'
           img(src="https://static.jboss.org/images/rhd/promo/RHDev_homepromo_summit2017_09may2017.png" alt="Summit Recap Img")
           span On-demand presentations, keynotes, and announcements.
-        a.spotlight-item href='#{site.base_url}/cheat-sheet/advanced-linux-commands-cheatsheet/'
-          img(src="https://static.jboss.org/images/rhd/promo/RHDev_homepromo_advlinuxcheat_11may2017.png" alt="Get the Advanced Linux Cheat Sheet Img")
-          span LAMP applications, SELinux Booleans and more.
+        a.spotlight-item href='#{site.base_url}/promotions/building-reactive-microservices-in-java'
+          img(src="https://design.jboss.org/redhatdeveloper/marketing/reactivemicroservices_book/website/images/RHDev_homepromo_reactivemicroservices_21apr2017.png.png" alt="Reactive Microservices Img")
```
Comment (LightGuard) [inline index.html.slim]: Are we no longer uploading images to static.jboss.org?

Comment (adelasofia) [inline index.html.slim]: I messaged James about this, we should wait for the link to be updated before merging

Comment (insectengine) [inline index.html.slim]: I hope I fixed this link correctly. Please let me know if I didn't.

**Pull request merged**